pkg/transaction/metric/metricscollector.go

```diff
@@ -167,15 +173,15 @@ func (c *collector) Execute() error {
 	c.metricEndTime = now()
 	c.orgGUID = c.getOrgGUID()
 	c.logger.
-		WithField("startTimestamp", util.ConvertTimeToMillis(c.usageStartTime)).
-		WithField("endTimestamp", util.ConvertTimeToMillis(c.usageEndTime)).
-		WithField("eventType", "usage").
+		WithField(startTimestamp, util.ConvertTimeToMillis(c.usageStartTime)).
+		WithField(endTimestamp, util.ConvertTimeToMillis(c.usageEndTime)).
+		WithField(eventType, "usage").
 		Debug("generating usage event")
 
 	c.logger.
-		WithField("startTimestamp", util.ConvertTimeToMillis(c.metricStartTime)).
-		WithField("endTimestamp", util.ConvertTimeToMillis(c.metricEndTime)).
-		WithField("eventType", "metric").
+		WithField(startTimestamp, util.ConvertTimeToMillis(c.metricStartTime)).
+		WithField(endTimestamp, util.ConvertTimeToMillis(c.metricEndTime)).
+		WithField(eventType, "metric").
```

Comment:
and here we are using some consts and some inline strings

Reply:
good point... fixing these too